ui/scss/component/_file-tile.scss

```diff
@@ -19,4 +19,20 @@ $height-file-tile: $spacing-vertical * 6;
   .card__title-primary {
     margin-top: 0;
   }
-}
\ No newline at end of file
+  .meta {
```

Comment:
I'm not sure we need to retain this if we match the behavior of discover, but if there is a new styling here specific to `file-tile__row`, we ought to make it a BEM class name and drop `meta`.

Reply:
> add experimental fileTile styles 87c4bd2
Ok, I'll remove this, was just a simple experiment :+1: